pkg/registry/defaults.go

```diff
@@ -34,7 +34,7 @@ const (
 
 	// Defaults/constants for devfile registry storages
 	DefaultDevfileRegistryVolumeSize = "1Gi"
-	DevfileRegistryVolumeEnabled     = true
+	DevfileRegistryVolumeEnabled     = false
```

Comment:
Can we also update the comment above `IsStorageEnabled` to say it returns false by default? https://github.com/devfile/registry-operator/blob/main/pkg/registry/defaults.go#L136.  Otherwise, everything looks good